standard/structs.md

```diff
@@ -476,3 +500,73 @@ Static constructors for structs follow most of the same rules as for classes. Th
 Automatically implemented properties ([§15.7.4](classes.md#1574-automatically-implemented-properties)) use hidden backing fields, which are only accessible to the property accessors.
 
 > *Note*: This access restriction means that constructors in structs containing automatically implemented properties often need an explicit constructor initializer where they would not otherwise need one, to satisfy the requirement of all fields being definitely assigned before any function member is invoked or the constructor returns. *end note*
+
+### §safe-context-rules Safe context constraint for ref struct types
+
+#### §safe-context-rules-general General
```

Comment:
I'm having a hard time reviewing this due to the change of terminology.  Then I realized all I had to do was... https://gist.github.com/gafter/d6068f35deb2a558b08b478f9dba1b1f (part 1...)

Reply:
Leaving this unresolved - we'll work out who will go through this gist later.